### PR TITLE
idr0062

### PR DESCRIPTION
**Pull request by jburel**
add prediction notebook, not using zarr

### CODE REVIEW

**Comment by dominikl** [comment]
Works fine on mybinder. Tried it running jupyter locally, failed installing stardist with 
```
error: Command "clang -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -fwrapv -O2 -Wall -fPIC -O2 -isystem /Users/dom/miniconda3/envs/jupyter/include -fPIC -O2 -isystem /Users/dom/miniconda3/envs/jupyter/include -I/private/var/folders/24/rc160jc95_q5g0pzs3tzc4wc0000gn/T/pip-build-env-sr1qv00b/overlay/lib/python3.10/site-packages/numpy/core/include -Istardist/lib/external/qhull_src/src -Istardist/lib/external/nanoflann -I/Users/dom/miniconda3/envs/jupyter/include/python3.10 -c stardist/lib/external/qhull_src/src/libqhull_r/libqhull_r.c -o build/temp.macosx-10.9-x86_64-3.10/stardist/lib/external/qhull_src/src/libqhull_r/libqhull_r.o -std=c++11" failed with exit status 1
```
But I guess that can be ignored.

**Comment by jburel** [comment]
I have a problem with the newer version of stardist that needs to be fixed before further testing

File: README.md [hunk 1]
```diff
@@ -29,6 +32,17 @@ cd omero-guide-python
 repo2docker .
 ```
 
+Building locally using conda and Jupyter:
+
+ * Create a conda environment i.e. ``conda env update -n stardist -f binder/environment.yml``
+ * Activate the newly created environment
+
+The following steps are only required if you want to the notebooks
```
Comment (dominikl) [inline README.md]: Something's missing here, "want to run the notebooks" maybe?

**Comment by jburel** [comment]
To test the updated instructions, you will need to use conda.

**Comment by dominikl** [comment]
👍 Looks good, tested locally using conda. There might just be a problem with the idr0062_prediction.ipynb notebook. The section "compare the labels" only shows labels on the right side image, the left side image looks like it's only the image itself. That does not happen on the idr0062_prediction_zarr_public_s3 notebook, there both images have labels.

**Comment by jburel** [comment]
@dominikl The problem with labels is now fixed.

**Comment by dominikl** [comment]
👍 Looks good now!

**Comment by jburel** [comment]
@pwalczysko are you okay if I merge this PR and add potentially more in follow-up work?

**Comment by pwalczysko** [comment]
> @pwalczysko are you okay if I merge this PR and add potentially more in follow-up work?

Cannot build it now, but I am sure it is okay, as @dominikl okayed it.

**Comment by dominikl** [comment]
I used conda, worked fine.

File: README.md [hunk 2]
```diff
@@ -29,6 +32,21 @@ cd omero-guide-python
...
 
+Building locally using conda and Jupyter:
+
+ * Create a conda environment i.e. ``conda env update -n stardist -f binder/environment.yml``
+ * Activate the newly created environment
+
+The following steps are only required if you want to run the notebooks
+ * If you have Anaconda installed:
+   * Start Jupyter from the Anaconda-navigator
+   * Select the notebook you wish to run and select the ``Kernel>Change kernel>Python [conda env:stardist]
```
Comment (pwalczysko) [inline README.md]: This is not formatted correctly, let me see if I can make suggestion

Comment (pwalczysko) [reply README.md]:
```suggestion
   * Select the notebook you wish to run and select the ``Kernel>Change kernel>Python [conda env:stardist]``
```

maybe like this ?

File: README.md [hunk 3]
```diff
@@ -29,6 +32,21 @@ cd omero-guide-python
 repo2docker .
 ```
 
+Building locally using conda and Jupyter:
+
+ * Create a conda environment i.e. ``conda env update -n stardist -f binder/environment.yml``
+ * Activate the newly created environment
+
+The following steps are only required if you want to run the notebooks
+ * If you have Anaconda installed:
```
Comment (pwalczysko) [inline README.md:41]: This does not seem logical cf. the paragraph below.

Comment (pwalczysko) [reply README.md:41]:
```suggestion
 * If you have Jupyter installed:
```

Maybe like this ?

Comment (jburel) [reply README.md:41]: Anaconda is correct

**Comment by pwalczysko** [comment]
After the switch of the kernel, the notebook worked fine for me too locally.

File: README.md [hunk 4]
```diff
@@ -29,6 +32,21 @@ cd omero-guide-python
...
+   * Start Jupyter from the Anaconda-navigator
+   * Select the notebook you wish to run and select the ``Kernel>Change kernel>Python [conda env:stardist]``
+ * If Anaconda is not installed:
+   * In the environment, install ``jupyter`` e.g. ``pip install jupyter``
+   * Add the virtualenv as a jupyter kernel i.e. ``ipython kernel install --name "stardist" --user``
+   * Open jupyter notebook i.e. ``jupyter notebook`` and select the ``stardist`` kernel if set or ``[conda env:stardist]`` if available
```
Comment (pwalczysko) [inline README.md]:
```suggestion
   * Open jupyter notebook i.e. ``jupyter notebook`` and select the ``stardist`` kernel or ``[conda env:stardist]`` according to what is available
```

Comment (pwalczysko) [reply README.md]: suggesting that above as was not sure why to use two wordings (``if set`` and ``if available``).
Reject if not appropriate

**Comment by pwalczysko** [comment]
As discussed, ready to merge fmpov, only one last reformulation suggested https://github.com/ome/omero-guide-python/pull/37#discussion_r840682701

**Pull request merged**